embedded-wallets/authentication/README.mdx

```diff
@@ -17,19 +17,19 @@ The SDKs are now branded as MetaMask Embedded Wallet SDKs (formerly Web3Auth Plu
 
 ::::
 
-## Supported Authentication Methods
+## Supported authentication methods
 
 Embedded Wallets supports a wide range of authentication options, enabling developers to meet the needs of diverse user bases and platform requirements.
 
-### Native Email & SMS Login
+### Native email & SMS log in
 
-Provides passwordless login via one-time passcodes (OTP)/ magic links sent to the user's [email address](/embedded-wallets/authentication/basic-logins/email-passwordless) or [phone number](/embedded-wallets/authentication/basic-logins/sms-otp). This method is widely accessible and especially useful for onboarding users unfamiliar with Web3 or crypto wallets.
```

Comment:
Minor comment: I think it makes sense to use "login" when referred to as a noun, and "log in" when used as a verb, WDYT? Should decide on this as this is probably used frequently throughout the EW docs.